dispatch/migrations/0001_initial.py

```diff
@@ -0,0 +1,293 @@
+# -*- coding: utf-8 -*-
```

Comment:
Since we haven't been committing migrations files (although we should maybe start), can you remove this file?